Marlin/Configuration_adv.h

```diff
@@ -1547,6 +1547,9 @@
     #if ANY(HAS_MARLINUI_U8GLIB, TFT_COLOR_UI)
       #define BOOT_MARLIN_LOGO_SMALL      // Show a smaller Marlin logo on the Boot Screen (saving lots of flash)
     #endif
+    #if ENABLED(TFT_COLOR_UI)
+      //#define BOOT_MARLIN_LOGO_USES_RLE16 // Uses less flash space.
+    #endif
```

Comment:
I'd move this to the existing `#if ENABLED(TFT_COLOR_UI)` block in `Configuration.h`

https://github.com/MarlinFirmware/Marlin/blob/60f591f3b6e06fdef903e10f81ac96f65e30d78e/Marlin/Configuration.h#L3288-L3299